File: Programming methods/listy/lista 5/z7.pl
split([],_,[],[]).
split([H|T],M,[H|S],B):-H<M,!,split(T,M,S,B).
split([H|T],M,S,[H|B]):-split(T,M,S,B).
qsor(X,S):-qsor(X,[],S).
qsor([],A,A).
qsor([H|T],A,E):-
	split(T,H,S,B),
	qsor(B,A,S1),
	qsor(S,[H|S1],E).

File: WWW course/lista7/z7.js
function listCookies() {
    var theCookies = document.cookie.split(';');
    var aString = '';
    for (var i = 1 ; i <= theCookies.length; i++) {
        aString += i + ' ' + theCookies[i-1] + "\n";
    }
    return aString;
}

/*ciastka:
mały fragment tekstu, który serwis internetowy wysyła do przeglądarki i który przeglądarka wysyła z powrotem 
przy następnych wejściach na witrynę. Używane jest głównie do utrzymywania sesji np. poprzez wygenerowanie 
i odesłanie tymczasowego identyfikatora po logowaniu. Może być jednak wykorzystywane szerzej poprzez 
zapamiętanie dowolnych danych, które można zakodować jako ciąg znaków. Dzięki temu użytkownik nie musi 
wpisywać tych samych informacji za każdym razem, gdy powróci na tę stronę lub przejdzie z jednej strony na inną.
*/ 